src/utils/transaction-guards.ts

```diff
@@ -124,8 +125,24 @@ export const isSignableBy = (txSummary: TransactionSummary, walletAddress: strin
   return !!executionInfo?.missingSigners?.some((address) => address.value === walletAddress)
 }
 
-export const isExecutable = (txSummary: TransactionSummary, walletAddress: string): boolean => {
-  if (!txSummary.executionInfo || !isMultisigExecutionInfo(txSummary.executionInfo)) {
+export const _isNextTx = (txId: string, items: TransactionListItem[]) => {
+  const hasNextLabel = items.find(isLabelListItem)?.label === LabelValue.Next
```

Comment:
Relying on a label seems hacky tbh.
Can’t we just check that it’s first in the queue and that the nonce is >= safe nonce?

Reply:
I didn't think of that - thanks! All we need to do is compare the nonce. I've adjusted it accordingly.